app/javascript/helpers/storage_manager/filter-resources-by-capabilities.js

```diff
@@ -0,0 +1,35 @@
+const equalsUnsorted = (arr1, arr2) => arr1.length === arr2.length
+  && arr2.every((arr2Item) => arr1.includes(arr2Item))
+  && arr1.every((arr1Item) => arr2.includes(arr1Item));
```

Comment:
Looks like this function is been used on a few other pages? is it possible to write it as a common function and export it?

Reply:
hi, turns out @agrare has already merged the provider repo part of this change, so now we have a mismatch between the way the capabilities are saved in the provider's models and the way the ui tries to parse them..
any chance we skip this comment and merge? if there's no other problem of course.

we're supposed to change this function anyway in a future PR, and could export it then.

thanks!

Reply:
done